main/externalData/balances/index.ts

```diff
@@ -17,15 +17,21 @@ export default function (store: Store) {
         .filter(n => (n.connection.primary || {}).connected || (n.connection.secondary || {}).connected)
     },
     getCustomTokens: () => (store('main.tokens.custom') || []) as Token[],
-    getKnownTokens: (address?: Address) => ((address && store('main.tokens.known', address)) || []) as Token[],
+    getKnownTokens: (address?: Address): Token[] => {
```

Comment:
I think we're probably gonna move this into a store action but in general it would be a little clearer to make this 2 separate functions. having functions with separate code paths just based on specific "magic" values of the parameters can be confusing and it may not be immediately obvious or intuitive why you need pass `undefined` in order to get all the known tokens